presto-main/src/main/java/com/facebook/presto/SessionRepresentation.java

```diff
@@ -313,7 +314,9 @@ public Session toSession(SessionPropertyManager sessionPropertyManager, Map<Stri
                         principal.map(BasicPrincipal::new),
                         roles,
                         extraCredentials,
-                        extraAuthenticators),
+                        extraAuthenticators,
+                        Optional.empty()),
+                ImmutableList.of(), // TODO: figure out if we need to store certificates.
```

Comment:
Is this TODO still applicable ?

Reply:
Logically it should work since this function is called to generate a task executor, which is after the permission check and the certificates should not be needed. But I am not 100% sure and so I left a todo here. I can remove it for now.

Reply:
Generally when leaving comment, we need to make it actionable for next developer. 

Something like currently this is an empty list, but in the later if this condition changes then it needs this action kind of it.